pkg/cli/zip_per_node.go

```diff
@@ -57,6 +65,27 @@ func makePerNodeZipRequests(
 	}
 }
 
+// Tables collected from each tenant SQL node in a debug zip.
+var debugZipTablesPerTenantNode = []string{
```

Comment:
Same comment here about sharing the overlapping subset in a single slice. Also, nit, they're ordered with the tenant later in the other file.